test/alphabet/nucleotide/nucl16_test.cpp

```diff
@@ -30,141 +30,233 @@
 // OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
 // DAMAGE.
 //
-// ============================================================================
-// Author: Sara Hetzel <sara.hetzel AT fu-berlin.de>
-// ============================================================================
-
-#include <array>
+// ==========================================================================
 
 #include <gtest/gtest.h>
+#include <sstream>
+
 #include <seqan3/alphabet/nucleotide/nucl16.hpp>
 
 using namespace seqan3;
+using namespace seqan3::literal;
 
 constexpr char capital_char[] = {'A', 'B', 'C', 'D', 'G', 'H', 'K', 'M', 'N', 'R', 'S', 'T', 'U', 'V', 'W', 'Y'};
 constexpr char lower_char[] = {'a', 'b', 'c', 'd', 'g', 'h', 'k', 'm', 'n', 'r', 's', 't', 'u', 'v', 'w', 'y'};
 constexpr nucl16 all_nucl16[] = {nucl16::A, nucl16::B, nucl16::C, nucl16::D, nucl16::G, nucl16::H, nucl16::K,
                                  nucl16::M, nucl16::N, nucl16::R, nucl16::S, nucl16::T, nucl16::U, nucl16::V,
                                  nucl16::W, nucl16::Y};
 
-TEST(alphabet_nucleotides_nucl16_test, default_ctr)
+// default/zero construction
+TEST(nucl16, ctr)
 {
-    constexpr nucl16 nucleotide{};
+    nucl16 t1;
+}
 
-    EXPECT_EQ(nucleotide, nucl16::A);
+// zero initialization
+TEST(nucl16, zro)
+{
+    nucl16 t0{};
+    EXPECT_EQ(t0, nucl16::A);
 }
 
-TEST(alphabet_nucleotides_nucl16_test, copy_ctr)
+// copy construction
+TEST(nucl16, cp_ctr)
 {
-    constexpr nucl16 nucleotide1{nucl16::B};
-    constexpr nucl16 nucleotide2{nucleotide1};
+    nucl16 t1{nucl16::C};
```

Comment:
I think the tests are super short and rather intuitive. Whether you call a variable `a` or `i` or `t0` or `t1` doesn't make much of difference.